_data/tiwaripramod99/tiwaripramod99.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Pramod Tiwari # No longer than 18 characters
+institution: U.V Patel College of Engineering # no longer than 58 characters
+profile_pic: tiwaripramod99.png # Name and extension of your profile picture(ex. mona.png)
+quote: People who are crazy enough to think that they can change the world are the one's who do. Programmers being one of those. # no longer than 100 characters
```

Comment:
Your quote does not meet the character length requirement (100 characters max, current character count: 121).

Reply:
Cannot make changes to this PR since it's already closed

Reply:
You can open a new PR

Reply:
I did, but the form has been closed. And I believe that's the reason it hasn't been reviewed yet.
Here's a link to the new PR.
https://github.com/education/graduation/pull/4179